CIP-0019/README.md

```diff
@@ -110,6 +110,9 @@ Some addresses (types 6 and 7) carry no delegation part whatsoever. Their associ
 
 ##### Pointers
 
+> **Note**
+> From the Conway ledger era, new pointer addresses cannot added to mainnet.
```

Comment:
```suggestion
> From the Conway ledger era, new pointer addresses cannot added to Mainnet.
```
`Mainnet` = capitalised because there's only one of them (like `Internet`)